ga_core/settings.py

```diff
@@ -26,10 +26,13 @@
 
     PROD_HOST = os.environ['PROD_HOST']
     DEBUG_HOST = os.environ['DEBUG_HOST']
+    CSRF_TRUSTED_ORIGINS = os.environ['CSRF_TRUSTED_ORIGINS']
```

Comment:
for form submission purpose. The value for this variable is http://localhost:1337 currently which is used during testing as the server is running on localhost. During prod, this might be whichever hostname:port / url the prod is running